src/swift/Data.swift

```diff
@@ -156,19 +193,41 @@ public struct DispatchData : RandomAccessCollection {
 	/// - parameter pointer: A pointer to the buffer you wish to copy the bytes into.
 	/// - parameter count: The number of bytes to copy.
 	/// - warning: This method does not verify that the contents at pointer have enough space to hold `count` bytes.
+	@available(swift, deprecated: 4, message: "Use copyBytes(to: UnsafeMutableRawBufferPointer, count: Int) instead")
 	public func copyBytes(to pointer: UnsafeMutablePointer<UInt8>, count: Int) {
 		_copyBytesHelper(to: pointer, from: 0..<count)
 	}
+
+	/// Copy the contents of the data to a pointer.
+	///
+	/// - parameter pointer: A pointer to the buffer you wish to copy the bytes into.
+	/// - parameter count: The number of bytes to copy.
+	/// - warning: This method does not verify that the contents at pointer have enough space to hold `count` bytes.
```

Comment:
This appears to duplicate the role of `UnsafeMutableRawBufferPointer.initializeMemory<S: Sequence>(as: S.Iterator.Element.Type, from source: S)`. There shouldn't be a performance penalty if we do the `_copyContents` trick (which it looks like `UnsafeMutableRawBufferPointer` isn't taking advantage of, but should). We should probably revisit this. cc @phausler